telemetry/telemetry_record.py

```diff
@@ -1,23 +1,30 @@
-"""Record WebPageReplay
+"""Record WebPageReplay, modify, and record page load times
 
 Records running a set of target urls on the remote tablet, modifies cacheable
 objects, then re-runs to see page load time differences.
 
-Usage: sudo python record_wpr.py
+Usage: sudo python telemetry_record.py
 
```

Comment:
It'd be helpful for understanding if the top of the file showed a little sketch of how the data flow works. e.g.

```
The script proceeds as follows:
 1. Read target_urls
 2. Write page_set.py
 3. Record wpr in /path/to/wpr
 4. Modify wpr, and save to /path/to/modified
 5. Run benchmarks, with results saved to /path/to/results
 6. Read in results, take the minimum page load time for each url, write final results to /foo